Jenkinsfile

```
@@ -35,19 +38,13 @@ pipeline {
                         options {
                             timeout(time: 60, unit: 'MINUTES')
                         }
-                        environment {
-                            DOCKERHUB_ORGANISATION = "${infra.isTrusted() ? 'jenkins' : 'jenkins4eval'}"
-                        }
                         stages {
                             stage('Prepare Docker') {
                                 when {
                                     environment name: 'IMAGE_TYPE', value: 'linux'
                                 }
                                 steps {
-                                    sh '''
-                                    docker buildx create --use
-                                    docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-                                    '''
+                                    sh 'make ci-init'
```

```suggestion
                                    sh 'make buildx-init'
```

nit: the intent is to ensure buildx is initialized, so proposal to (slightly) change the target. The keyword `init` is good and clear 👍